scripts/s3/s3api/bucket_operations_negative.py

```diff
@@ -0,0 +1,112 @@
+                    assert False, f"Able to see bucket with invalid bucket name. {resp}"
+                except bcexceptions.ClientError as err:
+                    # If a client error is thrown, then check that it was a 404 error.
+                    # If it was a 404 error, then the bucket does not exist.
+                    self.log.info('Error Code: {}'.format(err.response['Error']['Code']))
+                    self.log.info('Error Message: {}'.format(err.response['Error']['Message']))
+                    error_code = int(err.response['Error']['Code'])
+                    if error_code == 403:
+                        self.log.info("Private Bucket. Forbidden Access!")
+                    elif error_code == 404:
+                        self.log.info("Bucket Does Not Exist!")
+
+                try:
+                    self.log.info("upload_n_number_objects")
+                    resp = await self.upload_n_number_objects(bucket_name, file_size)
+                    assert False, f"Able to upload objects with invalid bucket. {resp}"
+                except (b3exceptions.S3UploadFailedError,
+                        bcexceptions.InvalidBucketName,
+                        bcexceptions.ParamValidationError) as err:
+                    self.log.info("Invalid bucket url: {%s}\nException: {%s}", self.s3_url, err)
+                    self.log.info('Error Code: {}'.format(err.response['Error']['Code']))
+                    self.log.info('Error Message: {}'.format(err.response['Error']['Message']))
+
+                try:
+                    self.log.info("List Objects in buckets")
+                    resp = await self.list_objects(bucket_name)
+                    assert False, f"Able to list objects with invalid bucket. {resp}"
+                except (bcexceptions.InvalidBucketName, bcexceptions.ParamValidationError) as err:
+                    self.log.info("Invalid bucket url: {%s}\nException: {%s}", self.s3_url, err)
+                    self.log.info('Error Code: {}'.format(err.response['Error']['Code']))
+                    self.log.info('Error Message: {}'.format(err.response['Error']['Message']))
+                try:
+                    self.log.info("Delete bucket %s with all objects in it.", bucket_name)
+                    resp = await self.delete_bucket(bucket_name, True)
+                    assert False, f"Able to list objects with invalid bucket. {resp}"
+                except (b3exceptions.ResourceNotExistsError, bcexceptions.ParamValidationError) as err:
+                    self.log.info("Invalid bucket url: {%s}\nException: {%s}", self.s3_url, err)
+                    self.log.info('Error Code: {}'.format(err.response['Error']['Code']))
+                    self.log.info('Error Message: {}'.format(err.response['Error']['Message']))
+
+                self.log.info("Iteration %s is completed of %s", iteration, self.session_id)
+            except Exception as err:
```

Comment:
Outer exception is not required as it is being used in negative call.

Reply:
Removed